packages/components/pie-input/src/index.ts

```diff
@@ -204,6 +203,7 @@ export class PieInput extends FormControlMixin(RtlMixin(LitElement)) implements
                 <input
                     type=${ifDefined(type)}
                     .value=${live(value)}
+                    id=${ifDefined(name)}
```

Comment:
It might be scope creep so feel free to say no, but could it be worth changing the `name` property to be required?

Reply:
I'm going to create a ticket around this change as it looks like the ID change here still doesn't link up properly with a labelled element if that's ok so for now im removing the change.